spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -76,104 +77,132 @@ public class StreamApplicationTests {
 }
 ----
 
-== Spring Cloud Stream Main Concepts
+== Main Concepts
 
-Spring Cloud Stream provides a number of abstractions and primitives that simplify writing message-driven microservices.
-In this section we will provide an overview of:
+Spring Cloud Stream provides a number of abstractions and primitives that simplify the writing of message-driven microservice applications.
+This section gives an overview of the following:
 
-* Spring Cloud Stream application model together with the Binder abstraction
-* Persistent publish-subscribe and consumer group support
-* Partitioning
-* Pluggable Binder API
+* A standard application model
```

Comment:
Maybe we can just drop 'standard' (in the sense that there's no non-standard alternative) and just say 'The application mode' ?